diff -rupN FreeRTOS_Base/tasks.c FreeRTOS_V8.1.2/tasks.c
--- FreeRTOS_Base/tasks.c	2015-04-08 12:10:22.604331000 +0200
+++ FreeRTOS_V8.1.2/tasks.c	2015-04-08 12:11:53.268331000 +0200
@@ -2111,7 +2111,7 @@ BaseType_t xSwitchRequired = pdFALSE;
 #endif /* configUSE_APPLICATION_TASK_TAG */
 /*-----------------------------------------------------------*/
 
-void vTaskSwitchContext( void )
+void __attribute__((used)) vTaskSwitchContext( void )
 {
 	if( uxSchedulerSuspended != ( UBaseType_t ) pdFALSE )
 	{
